client/src/components/common/names-capture.vue

```diff
@@ -70,16 +70,28 @@
                               :name="Math.random()"/>
               </v-col>
               <v-col cols="4" class="py-0" v-if="designationAtEnd">
-                <v-select :error-messages="des1Message"
-                          :hide-details="hide"
-                          :items="items"
-                          :menu-props="props"
-                          :value="nameChoices.designation1"
-                          @blur="handleBlur(); showDesignationErrors.des1 = true"
-                          @input="editChoices('designation1', $event, true)"
-                          filled
-                          id="designation-1-select"
-                          label="Designation" />
+                <v-tooltip top
+                  transition="fade-transition"
+                  content-class="top-tooltip designation-tooltip"
+                >
+                  <template v-slot:activator="{ on }">
+                    <div v-on="on">
+                      <v-select :error-messages="des1Message"
+                                :hide-details="hide"
+                                :items="items"
+                                :menu-props="props"
+                                :value="nameChoices.designation1"
+                                @blur="handleBlur(); showDesignationErrors.des1 = true"
+                                @input="editChoices('designation1', $event, true)"
+                                filled
+                                id="designation-1-select"
+                                label="Designation" />
+                    </div>
+                  </template>
+                  <p>Your company name must include a designation.
+                  There are no legal differences between these designations.
+                  You can choose whichever you prefer.</p>
+                </v-tooltip>
```

Comment:
Not sure a refactor needs to happen in this PR but all of the selectors are very similar. It could be a good instance where we could break this out into a mini sub component to avoid duplication. 
Maybe i'm overlooking some complexity tho! Just a thought :)

Reply:
I totally agree but it need more time

Reply:
Ok, fair enough!